components/brave_wallet/common/config.gni

```diff
@@ -10,4 +10,5 @@ declare_args() {
   brave_infura_project_id = ""
   enable_bitcoin_by_default = !is_ios
   enable_zcash_by_default = !is_ios && !is_android
+  enable_orchard = brave_channel == "nightly" || brave_channel == "development"
```

Comment:
this and  https://github.com/brave/brave-core/pull/22864/files#diff-9d8e31934812518cf74129a1ae4b4aeb1228eeb56506caaa537eece6d62adee1R6 should move to the other PR